.travis.yml

```diff
@@ -1,18 +1,21 @@
 language: python
 sudo: false
 python:
-  - 2.6
   - 2.7_with_system_site_packages
+  - 3.4
```

Comment:
Agreed. So long as it's no additional effort to support 3.3, I think we should do so. As soon as it becomes an effort, let's drop support.